.gitignore

```diff
@@ -156,3 +156,4 @@ $RECYCLE.BIN/
 
 # Mac desktop service store files
 .DS_Store
+Scripts/Generated/
```

Comment:
It's always good to leave one blank line at the end of the file. So that you know for the future :)